html5/document.html.slim

```diff
@@ -85,7 +85,7 @@ html lang=(attr :lang, 'en' unless attr? :nolang)
                     </svg>
                 h1 = header.title
               #content = content
-          .bd-links-wrapper
+          . bd-links-wrapper
```

Comment:
Expected whitespace?

Reply:
@slivas - please see Pierre's question about the white space.